app/src/lib/status.ts

```diff
@@ -59,15 +60,17 @@ export function hasConflictedFiles(
 }
 
 /**
- * Determine if we have a `ManualConflict` type
- * or conflict markers
+ * Determine if we have any conflict markers or if its been resolved manually
  */
-export function hasUnresolvedConflicts(status: ConflictedFileStatus) {
+export function hasUnresolvedConflicts(
+  status: ConflictedFileStatus,
+  manualResolution?: ManualConflictResolution
```

Comment:
we have to take into account a (possible) `manualResolution` if its a manual conflict. i think this points to a larger issue with manual conflict resolution state existing as a part of conflict state and not associated with the file status. 😕 and that we need a `none` type in the `ManualConflictMethodKind` enum.

i'd like to dive into this more, but i think this gets us where we need to be for now. ⏳

Reply:
i don't love this being optional `?`, but again having an explicit type for "no manual resolution" would help with that... if this was `ManualConflictResolution | null` then when callers go to pull the value out of the `Map<string, ManualConflictResolution>` of resolutions in `conflictState` then they have to check for `undefined` fist and pass in `null` instead.